in_text/text_00_parent.yml

```
@@ -0,0 +1,38 @@
+title: >-
+  XX # overall data release title here
+
+abstract: >-
+  XX # a description of this data release
+
+# ----supporting publications, source code, data releases----
+cross-cites:
+  -
+    authors: ["XX_NAME_1", "XX_NAME_2"]
+    title: >-
+        XX # title of cited item
+    pubdate: XXXX # year of publication for cited item
+    link: http://doi.org/YY.XXXXX/XXXXXX
+  -
+    authors: ["XX_NAME_1", "XX_NAME_2"]
+    title: >-
+        XX # title of cited item
+    pubdate: XXXX # year of publication for cited item
+    link: http://doi.org/YY.XXXXX/XXXXXX
+
+entities:
+  -
+    data-name: Parent metadata landing page
+    data-description: >-
+      This is the landing page for three other data release units, described in the attributes section
```

suggesting to remove because we don't describe them in the attributes section
```suggestion
      This is the landing page for XX other data release units
```